chapters/3-measurement.tex

```diff
@@ -852,15 +886,15 @@ \subsection{Clustering or stratifying a random sample or assignment}
 
 %-----------------------------------------------------------------------------------------------
 
-\section{Doing power calculations for research design}
+\subsection{Doing power calculations for research design}
 
 % sampling error, randomization noise and the need for power calcs
 Random sampling and treatment assignment are noisy processes:
 it is impossible to predict the result in advance.
 By design, we know that the exact choice of sample or treatment
 will be uncorrelated with our key outcomes,
 but this lack of correlation is only true ``in expectation'' --
-that is, across a large number of randomizations.
+that is, the correlation goes to zero as the sample size goes to inifinity.
```

Comment:
```suggestion
that is, the noise component of the correlation between randomization and outcome 
will only be zero on average across a large number of randomized trials.
```

Reply:
this part is about expectation over experiments, not the sample size of a single experiment